forge-gui/res/cardsfolder/upcoming/the_parting_of_the_ways.txt

```diff
@@ -0,0 +1,14 @@
+Name:The Parting of the Ways
+ManaCost:4 R R
+Types:Enchantment Saga
+K:Saga:3:DBDig,DBTimeTravel,DBDestroy
+SVar:DBDig:DB$ Dig | DigNum$ 5 | ChangeNum$ All | ChangeValid$ Card | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBRepeat | SpellDescription$ Exile the top five cards of your library. For each nonland card exiled this way, put a number of time counters on that card equal to its mana value. If it doesn't have suspend, it gains suspend.
+SVar:DBRepeat:DB$ RepeatEach | UseImprinted$ True | RepeatCards$ Card.IsRemembered+nonLand | Zone$ Exile | RepeatSubAbility$ DBPutCounter | SubAbility$ DBCleanup
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Imprinted | CounterType$ TIME | CounterNum$ X | SubAbility$ GiveSuspend
+SVar:GiveSuspend:DB$ Pump | Defined$ Remembered | KW$ Suspend | PumpZone$ Exile | Duration$ Permanent | SubAbility$ DBCleanup
```

Comment:
wrong/incomplete Defined

Reply:
still looks pretty awkward you're Pumping each loop when only the first will do anything now
I'd recommend adding `ClearRemembered` on the RepeatEach so you can save the extra effect too

Reply:
Would it be better to have the pumpall outside the loop ?

Reply:
well then you wouldn't be able to get rid of DBCleanup

Reply:
ok, just need to stop using Imprinted now and change the lines accordingly